proto/cosmos/group/v1beta1/types.proto

```diff
@@ -138,13 +139,14 @@ message Proposal {
   uint64 proposal_id = 1;
 
   // address is the group account address.
-  string address = 2;
+  string address = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
 
   // metadata is any arbitrary metadata to attached to the proposal.
   bytes metadata = 3;
 
   // proposers are the account addresses of the proposers.
-  repeated string proposers = 4;
+  // ?
+  repeated string proposers = 4 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
May I apply `cosmos.AddressString` annotation to repeated string type?

Reply:
I think so